packages/mobile/src/verify/VerificationInputScreen.tsx

```diff
@@ -131,29 +268,38 @@ class VerificationInputScreen extends React.Component<Props, State> {
             {t('input.body2')}
           </Text>
           <Text style={styles.codeHeader}>{t('input.codeHeader1')}</Text>
```

Comment:
Not to be too fancy but I believe you could do a map here of `[0,1,2]` and add one for the codeHeader to be more concise

Reply:
Ok will do

Reply:
this is an interesting question. i believe react diffs on what is returned from render, so assuming that the component was exactly the same with the same props, i think it would not rerender. i guess to ensure this we could create that array outside of the component, and then also give each coderow and text a key to make extra sure

i think in cases where you create inline functions and pass as props, it will rerender because the functions are different